src/Microsoft.Health.Dicom.Core/Features/Retrieve/AcceptHeaderHandler.cs

```diff
@@ -68,26 +68,42 @@ public AcceptHeader GetValidAcceptHeader(ResourceType resourceType, IReadOnlyCol
 
         List<AcceptHeaderDescriptor> descriptors = _acceptableDescriptors[resourceType];
 
+        AcceptHeader selectedHeader = null;
         // since these are ordered by quality already, we can return the first acceptable header
         foreach (AcceptHeader header in orderedHeaders)
         {
             foreach (AcceptHeaderDescriptor descriptor in descriptors)
             {
-                if (descriptor.IsAcceptable(header))
+                if (descriptor.IsAcceptable(header) && (selectedHeader == null || IsHigherPriorityTransferSyntax(header, selectedHeader)))
                 {
-                    return new AcceptHeader(
+                    selectedHeader = new AcceptHeader(
                         header.MediaType,
                         header.PayloadType,
                         descriptor.GetTransferSyntax(header),
                         header.Quality);
+
+                    continue;
                 }
             }
         }
 
+        if (selectedHeader != null)
+        {
+            _logger.LogInformation("Selected accept header {AcceptHeader} for retrieving {ResourceType}.", selectedHeader, resourceType);
+            return selectedHeader;
+        }
+
         //  none were valid
         throw new NotAcceptableException(DicomCoreResource.NotAcceptableHeaders);
     }
 
+    // if no quality provided prioritize returning original transfer syntax
+    private static bool IsHigherPriorityTransferSyntax(AcceptHeader header, AcceptHeader selectedHeader)
+    {
+        bool isQualityGreater = (header.Quality ?? 0.000) >= (selectedHeader.Quality ?? 0.000);
+        return (header.TransferSyntax.Value == DicomTransferSyntaxUids.Original && isQualityGreater);
```

Comment:
This naming is misleading: `Original` should really be called `Any`. If we want to prioritize the original, we should look up the original transfer syntax from the DB.

Reply:
Hmm, I am not seeing the difference. It is still saying the same without knowing the exact transfer syntax UID.

Reply:
This logic will prioritize the original syntax if the request specifies any transfer syntax for the media type via `*`. 

However, if we have an image stored in `Explicit VR Little Endian` and a caller sends the following in the `Accept` header:
- media type `application/dicom`, transfer syntax `JPEG 2000`, no qvalue
- media type `application/dicom`, transfer syntax `Explicit VR Little Endian`, no qvalue

Then this tie-breaking logic will pick `JPEG 2000` as the transfer syntax to return and we will transcode.

Reply:
Correct. The goal is not to avoid transcoding, it is to prioritize * when given in a list of media type with no q value